vowpalwabbit/reductions/epsilon_decay.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+#pragma once
+
+#include "reductions_fwd.h"
+#include "distributionally_robust.h"
+#include "scored_config.h"
+#include "array_parameters.h"
+
+namespace VW
+{
+namespace epsilon_decay
+{
+VW::LEARNER::base_learner* epsilon_decay_setup(VW::setup_base_i&);
+
+struct epsilon_decay_score : scored_config
+{
+  epsilon_decay_score() : VW::scored_config() {}
```

Comment:
I dont think this is necessary, can be replaced with default

```suggestion
  epsilon_decay_score() = default;
```